.github/workflows/pull-request-integ.yml

```diff
@@ -88,6 +88,7 @@ jobs:
       - name: Iceberg Nessie test
         uses: gradle/gradle-build-action@v2
         with:
+          cache-read-only: true
```

Comment:
> By default, the gradle-build-action will only write to the cache from Jobs on the default (main/master) branch. 

https://github.com/gradle/gradle-build-action#using-the-caches-read-only

so isnt it redundant to do this on PR workflows?

Reply:
ah, right - always forget about that.